runtime/manifest-parser.peg

```diff
@@ -105,7 +105,7 @@ ManifestStorage
   {
     items = optional(items, extractIndented, []);
     return {
-      kind: 'view',
+      kind: 'store',
```

Comment:
Store is the concrete storage system, handle is the pointer to that? Whereas before we called both a view?

Reply:
Yes, that's correct.

The `kind` is internal between the manifest-parser and manifest, so no manifest file changes are needed, in manifest one would still use `view` (though i hope we'll eventually rename them too):
```
view MyThings of [Thing] in 'data/things.json'
```

Reply:
Nice thanks! That makes sense. Maybe an issue for the manifest format work?

Reply:
filed: https://github.com/PolymerLabs/arcs/issues/938